docs/content/gettingstarted/firstworkflow.md

```diff
@@ -1,40 +1,47 @@
 +++
 title = "First Workflow from repository"
 weight = 1
-aliases = [
-    "/gettingstarted"
-]
 +++
```

Comment:
why removing the aliases ? it was nice to not have an empty page au the root level

Reply:
the page is not empty anymore